docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
     environment:
       - DB_USERNAME=$DB_USERNAME
       - DB_PASSWORD=$DB_PASSWORD
-      - DB_HOST=$DB_HOST
+      - DB_HOST=mysqldb

```

Comment:
please leave this one as $DB_HOST so that in any other place where DB_HOSt is used it will be aligned to environment value

Reply:
This is my change, I applied it already to team A to keep docker api container  use container domain name every time.

Reply:
as it's just dokcer-compose then ok